tools/gatsby.puml

```diff
@@ -0,0 +1,76 @@
+@startuml
+sprite $gatsby [64x64/16] {
+FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
```

Comment:
This file does not look right. Did you use an input picture with alpha-channels? The background is supposed to be white/`00` but is actually black/`FF`
![grafik](https://user-images.githubusercontent.com/6781173/117304856-7e1eeb80-ae7e-11eb-8641-d3e57743bfdf.png)

Reply:
Yeah I used a .png instead of jpgs. I fix this right away